app/models/efile_submission.rb

```diff
@@ -128,9 +128,15 @@ def generate_irs_address
 
   def generate_form_1040_pdf
     filename = "IRS 1040 - TY #{tax_return.year} - #{irs_submission_id}.pdf"
-
+    temp_1040_pdf = Irs1040Pdf.new(self).output_file
+    temp_8812_pdf = Irs8812Pdf.new(self).output_file
+    combined_temp_file = Tempfile.new(
```

Comment:
I wonder too if we should update the name of the filename to make it clear that it also includes the 8812. Although i think i recall that the ticket wanted to continue using the 1040 document type so maybe its okay to keep the naming conventions.